R/rows.R

```diff
@@ -775,12 +775,19 @@ rows_insert_prep <- function(con, x_name, y, by, lvl = 0) {
   out
 }
 
-rows_get_or_execute <- function(x, sql, returning_cols) {
+rows_get_or_execute <- function(x, sql, returning_cols, call = caller_env()) {
   con <- remote_con(x)
+  msg <- "Can't modify database table {.val {remote_name(x)}}."
   if (is_empty(returning_cols)) {
```

Comment:
e.g. here the `tryCatch()` could wrap both the `dbExecute()` and `the `dbGetQuery()`